lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -292,17 +401,30 @@ class MockTimers {
     }
   }
 
-  enable(timers = SUPPORTED_TIMERS) {
+  /**
+   * Enables the MockTimers replacing the native timers with the fake ones.
+   * @param {EnableOptions} options
+   */
+  enable(options = { timersToEnable: SUPPORTED_TIMERS, now: 0 }) {
     if (this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'MockTimers is already enabled!',
-      );
+      throw new ERR_INVALID_STATE('MockTimers is already enabled!');
+    }
+
+    if (NumberIsNaN(options.now)) {
+      throw new ERR_INVALID_ARG_VALUE('now', options.now, `epoch must be a positive integer received ${options.now}`);
     }
 
-    validateArray(timers, 'timers');
+    if (!options.now) {
+      options.now = 0;
```

Comment:
This mutates the user’s passed in options object; we should clone it first before assigning to it, or use variables instead of accessing options later.